src/bag3_digital/schematic/mux2to1_matched.py

```diff
@@ -43,10 +43,24 @@ def __init__(self, database: ModuleDB, params: Param, **kwargs: Any) -> None:
     def get_params_info(cls) -> Dict[str, str]:
         return dict(
             inv_params='Output inverter parameters',
-            tri_params='Tristate inverter parameters'
+            tri_params='Tristate inverter parameters',
+            sel_params='Select inverter params',
         )
 
-    def design(self, inv_params: Param, tri_params: Param) -> None:
+    @classmethod
+    def get_default_param_values(cls):
```

Comment:
Add return type (Mapping[str, Any]). Pycharm's auto-complete does this automatically while writing the method name.